sdk/search/Azure.Search.Documents/tests/Utilities/SearchTestBase.cs

```diff
@@ -252,5 +252,35 @@ static void AssertEqual(object e, object a, string path)
                 delay = TimeSpan.FromSeconds(Math.Min(delay.TotalSeconds * 2, maxDelay.TotalSeconds));
             }
         }
+
+        /// <summary>
+        /// Wait until the document count for a given search index has crosssed
+        /// a minimum value.  This only does simple linear retries for a fixed
+        /// number of attempts.
+        /// </summary>
+        /// <param name="searchClient">Client for the index.</param>
+        /// <param name="minimumCount">Minimum document count to verify indexing.</param>
+        /// <param name="attempts">Maximum number of attempts to retry.</param>
+        /// <param name="delay">Delay between attempts.</param>
+        /// <returns>A <see cref="Task"/> to await.</returns>
+        protected async Task WaitForDocumentCountAsync(
```

Comment:
👍 I should use this in quite a few other places that have been a bit...unstable...for nightlies.